README.md

```diff
@@ -242,26 +242,49 @@ Vault support is currently a work-in-progress.
 - `token`: string. Auth token for the vault.
 
 # Usage
-Cashier comes in two parts, a [cli](cmd/cashier) and a [server](cmd/cashierd).  
+Cashier comes in two parts, a [cli](cmd/cashier) and a [server](cmd/cashierd).
 The server is configured using a HCL configuration file - [example](example-server.conf).
 
 For the server you need the following:
 - A new ssh private key. Generate one in the usual way using `ssh-keygen -f ssh_ca` - this is your CA signing key. At this time Cashier supports RSA, ECDSA and Ed25519 keys. *Important* This key should be kept safe - *ANY* ssh key signed with this key will be able to access your machines.
 - OAuth (Google or GitHub) credentials. You may also need to set the callback URL when creating these.
 
 ## Using cashier
-Once the server is up and running you'll need to configure your client.  
-The client is configured using either a [HCL](https://github.com/hashicorp/hcl) configuration file - [example](example-client.conf) - or command-line flags.  
+Once the server is up and running you'll need to configure your client.
+The client is configured using either a [HCL](https://github.com/hashicorp/hcl) configuration file - [example](example-client.conf) - or command-line flags.
+
+- `ca`          CA server (default "http://localhost:10000").
+- `config`      Path to config file (default "/Users/kevin/.cashier.conf").
+- `key_size`    Key size. Ignored for ed25519 keys (default 2048).
+- `key_type`    Type of private key to generate - rsa, ecdsa or ed25519 (default "rsa").
+- `public_file_prefix` Prefix for filename for public key and cert (optional, no default). The public key is put in a file with `.pub` appended to it; the public cert file in a file with `-cert.pub` appended to it.
+- `validity`    Key validity (default 24h0m0s).
```

Comment:
I know these came from the usage string, but maybe shorten this to just `24h` for easier reading.